bspump/http/web/components/field.py

```diff
@@ -57,4 +57,6 @@ def html(self, default=""):
         )
 
     def get_params(self, default="") -> dict:
-        return {self.name: {"type": str(type(self)), "description": self.description}}
+        if not default:
+            default = self.default
```

Comment:
Where is default used?

Reply:
I don't know how this `default` parameter got there in the first place, but it appears it was never used. Removed